assets/javascripts/discourse/components/modal/post-event-builder.js

```diff
@@ -5,6 +5,7 @@ import { inject as service } from "@ember/service";
 import { extractError } from "discourse/lib/ajax-error";
 import { cook } from "discourse/lib/text";
 import Group from "discourse/models/group";
+import I18n from "I18n";
```

Comment:
the new/recommended module path is now:

```suggestion
import I18n from "discourse-i18n";
```